<Feature: Family Setup>

empireboy opened the pull request.

projojoboy committed:
<Feature: Ingame UI> <Status: In progess>
Add Textmeshpro package
Add Sprite ui lowerbar
Add New font style
Add Stopwatch

projojoboy committed:
<Feature: Ingame UI> <Status: In progess>
Add Pause menu script

projojoboy committed:
Merge branch 'development' into feature/ingame-ui

# Conflicts:
#	Assets/Scripts.meta

projojoboy committed:
<Feature: Ingame UI> <Status: In progess>
Add Static properties class
Add Pause menu prototype
Add Static UIApi class
Add Static gameobjectevent class

Commit:
<Feature: Ingame UI> <Status: In progess>
Add Resume/Home button sprites
Add Level completion screen sprites
Add Temp art to pause screen

projojoboy committed:
<Feature: Ingame UI> <Status: In progess>
Add Properties class
Add Backend spawning windows

Remove PauseButton class

empireboy committed:
Merge branch 'feature/ingame-ui' of https://github.com/PVB-Team-1/ProefProeveTeam1 into family-setup

# Conflicts:
#	Assets/Scripts/Features.meta
#	Assets/Scripts/Features/UI.meta

projojoboy committed:
<Feature: Ingame UI> <Status: In progess>
Add Open&Close pause window

empireboy committed:
Merge branch 'feature/ingame-ui' of https://github.com/PVB-Team-1/ProefProeveTeam1 into family-setup

empireboy committed:
<Feature: Family Setup> <Status: Done>
Add Family class in Properties
Add Corresponding test files
Add Family class
Add Person class
Add FamilySetupWindow class
Add PersonSetupWindow class

empireboy committed:
<Feature: Family Setup> <Status: Done>
Add XML comments to the Properties class

empireboy committed:
Merge branch 'development' into feature/family-setup

empireboy committed:
<Feature: Family Setup> <Status: Done>
Add CloseLastWindow now removes the last window from the list
Add CloseAllWindows now clears the entire window list

empireboy committed:
Revert "Merge branch 'feature/ingame-ui' of https://github.com/PVB-Team-1/ProefProeveTeam1 into family-setup"

This reverts commit 6f4aef961d7bd6ecad5e99c6533f1cf65e36dc11, reversing
changes made to 9309725198bc7c4a41522c7eb4505f7644f5cc4f.

empireboy committed:
Revert from ingame-ui

empireboy committed:
<Feature: Family Setup> <Status: In progress>
Add Scripts from ingame-ui
Add PersonSetupWindow prefab
Add FamilySetupWindow prefab

empireboy committed:
<Feature: Family Setup> <Status: Done>
Add XML comments in UIApi
Add XML comments in WindowTypes

empireboy committed:
<Feature: Family Setup> <Status: Done>
Add FamilySetupWindow
Add PersonSetupWindow

empireboy committed:
<Feature: Family Setup> <Status: Done>
Remove FamilySetupTest

projojoboy commented on Assets/Scripts/Backend/Api/UIApi.cs:
```diff
@@ -0,0 +1,65 @@
+    public static List<GameObject> OpenedWindows = new List<GameObject>();
+
+	/// <summary>
+	/// Opens a window.
```
Try to specify summaries more, "Opens a window." Which window does it open? A random window?

projojoboy commented on Assets/Scripts/Properties/Family.cs:
```diff
@@ -0,0 +1,26 @@
+	/// <summary>
+	/// The name of the family.
+	/// </summary>
+	public string name;
```
Instead of "Name" I should've named this something like "FamilyName" to specify it a little bit more

projojoboy commented on Assets/Scripts/Properties/LoadWindows.cs:
```diff
@@ -0,0 +1,29 @@
+﻿using UnityEngine;
+using System.Runtime.CompilerServices;
+
```
Add xml comments

projojoboy reviewed (changes requested):
Just some missing XML comments

empireboy committed:
<Feature: Family Setup> <Status: Done>
Add LoadWindows with checks if the windows get loaded correctly
Add Inspector button in LoadWindows to automatically generate the windows array
